src/Controller/AventController.php

```diff
@@ -10,69 +10,77 @@
-            '09-utilisation-de-twig-en-phase-de-prototypage' => 'Avent/day_2017_09.html.twig',
-            '08-symfony-flex-la-nouvelle-facon-de-developper-avec-symfony' => 'Avent/day_2017_08.html.twig',
-            '07-decoupler-form-et-votre-modele' => 'Avent/day_2017_07.html.twig',
-            '06-headless-is-more' => 'Avent/day_2017_06.html.twig',
-            '05-deboguer-php-avec-gdb' => 'Avent/day_2017_05.html.twig',
-            '04-tagger-son-application' => 'Avent/day_2017_04.html.twig',
-            '03-deployer-un-projet-symfony-flex-sur-heroku' => 'Avent/day_2017_03.html.twig',
-            '02-la-nouvelle-configuration-par-defaut-de-symfony4' => 'Avent/day_2017_02.html.twig',
-            '01-easyadminbundle-l-arriere-guichet-easy-peasy' => 'Avent/day_2017_01.html.twig',
+            '24-unpack-tes-packs' => 'Avent/2017/day_24.html.twig',
+            '23-comment-l-architecture-hexagonale-nous-a-facilite-la-vie' => 'Avent/2017/day_23.html.twig',
+            '22-log-me-tender' => 'Avent/2017/day_22.html.twig',
+            '21-symfony-et-websockets' => 'Avent/2017/day_21.html.twig',
+            '20-elasticsearch-6-et-symfony-4' => 'Avent/2017/day_20.html.twig',
+            '19-composant-workflow-par-l-exemple' => 'Avent/2017/day_19.html.twig',
+            '18-structurer-sa-demarche-de-test' => 'Avent/2017/day_18.html.twig',
+            '17-bien-demarrer-avec-symfony-et-react' => 'Avent/2017/day_17.html.twig',
+            '16-des-astuces-de-securite-avec-symfony' => 'Avent/2017/day_16.html.twig',
+            '15-comment-ne-pas-trop-exposer-symfony' => 'Avent/2017/day_15.html.twig',
+            '14-une-api-graphql-avec-symfony' => 'Avent/2017/day_14.html.twig',
+            '13-creer-une-application-cross-platform' => 'Avent/2017/day_13.html.twig',
+            '12-serverless-php-et-symfony' => 'Avent/2017/day_12.html.twig',
+            '11-non-a-la-doctrine-decouvrez-pomm-et-re-decouvrez-postgresql' => 'Avent/2017/day_11.html.twig',
+            '10-comment-mettre-les-mains-dans-un-projet-existant' => 'Avent/2017/day_10.html.twig',
+            '09-utilisation-de-twig-en-phase-de-prototypage' => 'Avent/2017/day_09.html.twig',
+            '08-symfony-flex-la-nouvelle-facon-de-developper-avec-symfony' => 'Avent/2017/day_08.html.twig',
+            '07-decoupler-form-et-votre-modele' => 'Avent/2017/day_07.html.twig',
+            '06-headless-is-more' => 'Avent/2017/day_06.html.twig',
+            '05-deboguer-php-avec-gdb' => 'Avent/2017/day_05.html.twig',
+            '04-tagger-son-application' => 'Avent/2017/day_04.html.twig',
+            '03-deployer-un-projet-symfony-flex-sur-heroku' => 'Avent/2017/day_03.html.twig',
+            '02-la-nouvelle-configuration-par-defaut-de-symfony4' => 'Avent/2017/day_02.html.twig',
+            '01-easyadminbundle-l-arriere-guichet-easy-peasy' => 'Avent/2017/day_01.html.twig',
         ),
+        2019 => array(
+            '01-test' => 'Avent/2019/day_01.html.twig',
```

Comment:
😇 https://github.com/afsy/website/issues/162#issuecomment-555499528

Reply:
So you should tell me how you want to split blocks will markdown only :)